src/views/SubjectAllQuestions/Question/Question.tsx

```diff
@@ -108,6 +133,25 @@ export const Question = ({
                 <AddedByZdajSeIcon className={classes.icon} />
               </Tooltip>
             )}
+            {learntButtonData && (
+              <Tooltip title="Oznacz jako nauczone" placement="bottom">
+                <Button
+                  fullWidth
+                  component="label"
+                  variant="outlined"
+                  className={classes.learntWrapper}
+                >
+                  <SchoolIcon fontSize="small" />
+                  <Checkbox
```

Comment:
There still seems to be a reminiscent of the ripple on hover
![CleanShot 2023-05-30 at 22 22 00](https://github.com/bibixx/zdaj-se-pjatk/assets/14880213/1369af92-6155-4fd6-8fb3-d86d2106566f)